ginivision/src/main/java/net/gini/android/vision/internal/camera/photo/Exif.java

```diff
@@ -224,10 +224,46 @@ public Exif build() {
 
         private void addStringExif(@NonNull TiffOutputDirectory outputDirectory,
                 @NonNull TiffField field) throws ImageReadException {
-            byte bytes[] = field.getStringValue().getBytes(Charset.forName("US-ASCII"));
+            byte bytes[] = getStringFieldValue(field).getBytes(Charset.forName("US-ASCII"));
             addStringExif(outputDirectory, field.getTagInfo(), bytes);
         }
 
+        @NonNull
+        private String getStringFieldValue(@NonNull final TiffField field) throws ImageReadException {
```

Comment:
Could it be replaced with a toString or similar method on TiffField class?

Reply:
Unfortunately not since we don't initialize the instance, we receive it from `JpegImageMetadata#findEXIFValue()`.

By analysing the implementation again I discovered, that handling make and model tags as strings is not needed, we can simply add them like the other ones and eliminate a potential source of bugs completely.